files/jquery.csv-calc/update.json

```diff
@@ -0,0 +1,10 @@
+{
+    "packageManager": "github",
+    "name": "jquery.csv-calc",
+    "repo": "sutara79/jquery.csv-calc",
+    "files": {
+        "include": [
+            "jquery.csv-calc.min.js"
```

Comment:
That file is not located in root. You need need to set the basepath to dist/
https://github.com/sutara79/jquery.csv-calc/tree/1.0.2/dist